index.html

```diff
@@ -374,6 +379,11 @@ <h4>ASOCIAȚIA OPORTUNITĂȚI ȘI CARIERE</h4>
         <p>Privacy — Terms</p>
       </div>
     </footer>
+    <a href="#top-redirect">
+      <button class="top-redirect-btn">
+        <img src="./assets//arrow-up.svg" alt="" />
```

Comment:
The image somehow has a gray background. Please remove it.
Also, please make sure the image is showing on the page - at max width 1440px. 
If we are at the very top of the page, the arrow should be hidden, since there's no need to go up.